src/main/scala/scorex/core/validation/ValidationSettings.scala

```diff
@@ -0,0 +1,33 @@
+package scorex.core.validation
+
+import scorex.core.validation.ValidationResult.Invalid
+
+/**
+  * Configuration of validation.
```

Comment:
The sentence above could be skipped without any loss, as it is saying literally nothing in addition to the class name.